h/static/scripts/annotator/plugin/test/pdf-metadata-test.js

```diff
@@ -9,19 +9,23 @@ describe('pdf-metadata', function () {
 
     var event = document.createEvent('Event');
     event.initEvent('documentload', false, false);
+    fakeApp.url = 'fakeUrl';
```

Comment:
I think it makes tests more useful as documentation when the fake data at least looks like the real data, so eg. `http://fake.com` instead of `fakeUrl`.